test/cppsuite/test_harness/test.h

```diff
@@ -91,10 +91,13 @@ class test : public database_operation {
     checkpoint_manager *_checkpoint_manager = nullptr;
     runtime_monitor *_runtime_monitor = nullptr;
     thread_manager *_thread_manager = nullptr;
-    timestamp_manager *_timestamp_manager = nullptr;
     workload_generator *_workload_generator = nullptr;
+
+    protected:
+    timestamp_manager *_timestamp_manager = nullptr;
```

Comment:
Can we move those to the protected section line 86?

Reply:
This and the following comments addressed in [c8a93c7](https://github.com/wiredtiger/wiredtiger/pull/7872/commits/c8a93c78f86ad6acd13577aa0433aa1531408e22)